test/llvm_autogenerated/llvm-to-s.py

```diff
@@ -71,7 +75,8 @@ def main(args):
         # generate .s files.
         if cmd[0] != 'llc':
             continue
-        cmd[0] = os.path.join(BIN_DIR, cmd[0])
+        cmd[0] = os.path.join(bin_dir, cmd[0])
+        print ' '.join(cmd)
```

Comment:
Yes but I like for my scripts to tell me when they do things, and what they do :)